### PR TITLE
Ear 636 bypass blocks of content 

### PR DESCRIPTION
**Pull request by jasehumphr**
> ### BEFORE MAKING YOUR PR
>
> Please ensure:
>
> - There are no linting errors, all tests must pass
> - PR is named after JIRA ticket number e.g. EAR-###
> - **Accesibility** checks are completed:
>   - Elements have discernible and consistent focus states
>   - Elements can be navigated to and used by just a keyboard
>   - Elements and text can be read out by a screen reader, and the descriptions are understandable
> - Your feature / bug fix works across **GCP** and **AWS** (where appropriate)
>   - Are modifications to eq-publisher-v3 required?
>   - Are modifications to the Firestore data source required?

---

### What is the context of this PR?

> Describe what you have changed as part of this task and why; include a link to the Trello card/GitHub issue, related pull requests, etc as required.
>
> E.g. _"Currently, if you add a new question to Author the entire app crashed. This is because..."_

### How to review

> Add to the list below as appropriate, including screenshots when necessary

- Import the **Capability examples** questionnaire from pre-prod Author into your local environment and:
  - ensure it can be opened in Author;
  - then, ensure it can be viewed in Runner by pressing the **view survey** button
- Does this require a migration? We need one if existing JSON schema properties change

### What to do after everything is green

1. - [ ] Bring this branch up-to-date with Origin/Master
2. - [ ] If there are a lot of commits or some are not helpful to read, squash them down
3. - [ ] Click the **Merge** button on this pull request
4. - [ ] Does this change mean the **Capability examples** questionnaire should be updated?
5. - [ ] Move the Jira ticket for this task into the next stage of the process

### CODE REVIEW

File: eq-author/src/components/EditorLayout/Header/index.js [hunk 1]
```diff
@@ -59,7 +65,7 @@ export const UnconnectedHeader = (props) => {
 
   return (
     <>
-      <StyledHeader>
+      <StyledHeader id="SuperNav-4" tabIndex="-1" className="keyNav">
```
Comment (martyncolmer) [inline eq-author/src/components/EditorLayout/Header/index.js:68]: id not needed

File: eq-author/src/components/EditorLayout/Tabs/index.js [hunk 2]
```diff
@@ -48,16 +48,19 @@ const TABS = [
   {
     key: "design",
     children: "Design",
+    id: "SuperNav-3",
```
Comment (martyncolmer) [inline eq-author/src/components/EditorLayout/Tabs/index.js:51]: not needed

File: eq-author/src/App/pageskip/PageSkipPage.js [hunk 3]
```diff
@@ -0,0 +1,106 @@
+import React from "react";
```
Comment (martyncolmer) [inline eq-author/src/App/pageskip/PageSkipPage.js:1]: needs a better name link KeyboardShortcutsPage

File: eq-author/src/App/sharing/SharePage.js [hunk 4]
```diff
@@ -3,7 +3,7 @@ import CustomPropTypes from "custom-prop-types";
 
 import Header from "components/EditorLayout/Header";
 import ScrollPane from "components/ScrollPane";
-
+import styled from "styled-components";
```
Comment (martyncolmer) [inline eq-author/src/App/sharing/SharePage.js:6]: This import is not used

**Review by martyncolmer — CHANGES_REQUESTED** [review]
On the design page, the tab goes to the comments pane, but there is nothing there.

File: eq-author/src/App/keyboardShortcuts/KeyboardShortcutsPage.js [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+  text-align: left;
+  font-size: 1.25em;
+`;
+
+const TableDataB = styled.td`
+  padding: 1em;
+  text-align: left;
+  font-size: 1.25em;
+  border-top: 1px ${colors.grey35};
+  border-bottom: 1px ${colors.grey35};
```
Comment (farres1) [inline eq-author/src/App/keyboardShortcuts/KeyboardShortcutsPage.js:53]: These lines aren't visible as they aren't `solid`. Changing them to use `solid` displays them differently to the prototype as they don't touch the edges of the table.

File: eq-author/src/App/keyboardShortcuts/KeyboardShortcutsPage.js [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+  font-size: 1.25em;
+`;
+
+const TableData = styled.td`
+  padding: 1em;
+  text-align: left;
+  font-size: 1.25em;
+`;
+
+const TableDataB = styled.td`
```
Comment (farres1) [inline eq-author/src/App/keyboardShortcuts/KeyboardShortcutsPage.js:48]:
```suggestion
const TableDataDarkGrey = styled.td`
```
Suggested name change

File: eq-author/src/App/keyboardShortcuts/KeyboardShortcutsPage.js [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+`;
+
+const TableHeader = styled.td`
+  width: 50%;
+  padding: 1em;
+  text-align: left;
+  font-size: 1.25em;
+`;
+
+const TableData = styled.td`
```
Comment (farres1) [inline eq-author/src/App/keyboardShortcuts/KeyboardShortcutsPage.js:42]:
```suggestion
const TableDataLightGrey = styled.td`
```
Suggested name change

**Review by farres1 — CHANGES_REQUESTED** [review]
It looks really good! I've added some comments in the review

**Review by martyncolmer — APPROVED** [review]

**Pull request merged**